### PR TITLE
ready to install ts2dart into the angular build

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -115,7 +116,10 @@ var CONFIG = {
   transpile: {
     src: {
       js: ['modules/**/*.js', 'modules/**/*.es6'],
-      dart: ['modules/**/*.js']
+      dart: ['modules/**/*.js'],
+      // Migrating to TypeScript, one package at a time.
+      // See https://docs.google.com/document/d/14RJLhu6uuv7NchFkAb6PKzOOO0L7l3Z507eKWzkEUhQ/edit
+      ts2dart: ['modules/angular2/src/di/*.js', 'modules/angular2/test/di/*.js', 'modules/angular2/src/test_lib/*.js']
```
Comment (mprobst) [inline gulpfile.js]: Are we going to include the output of this in the actual production build?

File: gulpfile.js [hunk 2]
```diff
@@ -115,7 +116,10 @@ var CONFIG = {
   transpile: {
     src: {
       js: ['modules/**/*.js', 'modules/**/*.es6'],
-      dart: ['modules/**/*.js']
+      dart: ['modules/**/*.js'],
+      // Migrating to TypeScript, one package at a time.
+      // See https://docs.google.com/document/d/14RJLhu6uuv7NchFkAb6PKzOOO0L7l3Z507eKWzkEUhQ/edit
```
Comment (mprobst) [inline gulpfile.js]: Maybe create a g.co or goo.gl link?

**Comment by mprobst** [comment]
I think you'll have to rebase this, this has 9 commits going into it. And squash my commit, I presume that's the problem with the CLA bot.

**Comment by alexeagle** [comment]
No, not included in either dist/dart or dist/js, so I think we are throwing
the output away at present.
Squashed the commits and rebased, but there is a test failure so we need a
fix in the transpiler first...

**Comment by alexeagle** [comment]
This was merged in 43f4374944a2b7c7933968782ec0b362f93bd2fd

**Pull request closed without merging**